.github/labeler.yml

```diff
@@ -15,243 +15,530 @@
 # something else (e.g. CI if the PR also touches CI jobs).
 #
 # N.B. any-glob-to-all-files is misnamed; it acts like one-glob-to-all-files.
-# Therefore, to get any-glob-to-all-files semantics, there must be a single glob
-# with all matching patterns within braces.
+# Therefore, to do matching over multiple globs, they must be combined to a
```

Comment:
The documentation for labeler is very ambiguous, especially as it pertains to matching. "matching over multiple globs" is in my opinion more ambiguous than the original wording, which tries to maintain a consistency that makes it easier to understand how matching happens. All that "matching over multiple globs" means is that more than one glob is involved, somehow.

Reply:
Yes, I see. Now there the problem that the word glob is not well defined. What do we consider to be a glob: filename expansion, or is brace expansion also included? Also, the original wording is a bit problematic, as it might be contradictory. It says to have any-glob-to-all-files semantics - thus: any of the *multiple* globs to all files - there must be a *single* glob.

I have updated the wording again in an attempt to sidestep these problems.